### ISSUE TITLE
Rename "cwmsInterval" Config Sensor Property to "CwmsInterval"

### ISSUE THREAD

**Issue by jbatterman** [opendcs/opendcs#680]
**Describe the bug**
Not a bug, just a consistency error.  Other CWMS pathname properties are named with camel case (i.e., capital "C" on "Cwms" and then a capitalized first letter of the second word).  For example, "CwmsDuration", "CwmsParamType", "CwmsVersion".

**To Reproduce**
1. Go to DECODES Database Editor
2. Click on a Config with defined Sensors
3. Select a Sensor and choose "Edit Sensor"
4. See Properties table at the bottom of the GUI.

**Expected behavior**
For consistency, all CWMS pathname properties should be named "Cwms...".

**Screenshots**
![image](https://github.com/user-attachments/assets/ae3cb357-df60-4945-a9c0-b132227c45cc)

**Desktop/ServerSide (please complete the following information):**
 - OS: Windows Version 7.0.13-RC05
 - Version (server) 7.0.13-RC05

**Comment by MikeNeilson** [comment]
I agree with this, however I think we'll need to include both and a warning; assuming a case sensitive retrieval is used (may not which makes things easier). Going to take a bit to make sure all the conditions are right not to break existing setups.
